packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -253,6 +254,12 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
           // TODO(jacobr): this line is ascii art that it would be nice to
           // handle a little more generically in GUI debugging clients in the
           // future.
+          if (node.debugCreator?.element != null) {
+            yield node.debugCreator?.element?.describeOwnershipChain('Elements chain');
+            yield node.debugCreator?.element?.describeFirstLocalWidget('First user created widget');
+          } else {
+            yield describeForError('Elements chain is not available');
```

Comment:
Lets avoid all of the error messages that say "Elements chain is not available" everywhere as it is not actionable or related to the error the user saw.